src/components/header/header-progress/header-progress.styles.ts

```diff
@@ -16,7 +16,7 @@ export const useHeaderProgressStyles = createUseStyles(({ colors, typography })
     position: 'relative',
     height: formatSize(4),
     borderRadius: formatSize(4),
-    backgroundColor: colors.lines
+    backgroundColor: colors.progressLine
```

Comment:
Just update `lines` colors value for dark mode.
https://www.figma.com/file/c8fWgwUllZAfgEv4slZdgj/Temple-UI-Kit-v1?node-id=7%3A39162

Vova (designer) just have fixed it.